lib/cli.js

```diff
@@ -63,7 +63,7 @@ function translateOptions(cliOptions) {
         cache: cliOptions.cache,
         cacheFile: cliOptions.cacheFile,
         cacheLocation: cliOptions.cacheLocation,
-        fix: cliOptions.fix && (cliOptions.quiet ? quietFixPredicate : true),
+        fix: (cliOptions.fix || cliOptions.fixDryRun) && (cliOptions.quiet ? quietFixPredicate : true),
```

Comment:
Can you add a fatal error if `fix` and `fixDryRun` are used together? Currently, `--fix --fix-dry-run` is effectively the same as `--fix`, which is likely to be confusing.

Reply:
Is there an example of a fatal error on the configuration handling stage that I could refer to? Or should I just throw in `translateOptions`?

Reply:
I think something like [this](https://github.com/eslint/eslint/blob/439e8e697f761bebeb3dab085ba85139829397b2/lib/cli.js#L175-L176) would be fine.